internal/controllers/clusterclass/clusterclass_controller.go

```diff
@@ -73,7 +73,6 @@ func (r *Reconciler) SetupWithManager(ctx context.Context, mgr ctrl.Manager, opt
 	predicateLog := ctrl.LoggerFrom(ctx).WithValues("controller", "clusterclass")
 	err := ctrl.NewControllerManagedBy(mgr).
 		For(&clusterv1.ClusterClass{}).
-		Named("clusterclass").
```

Comment:
this is unrelated I assume?

Reply:
Yeah absolutely unrelated. Just saw it when doing some research around controller names while I had this branch open. I can open a separate PR if necessary